Source/Resources/Content/MSGraph/DeviceManagement/DeviceConfigurations/Security - Windows Firewall Notifications Disabled.json

```diff
@@ -8,7 +8,7 @@
       }
     }
   ],
-  "description": "Suppresses Windows Firewall alerts to the end user. This is applied so that users are not warned that the Domain and Private network firewall profiles allow inbound traffic by default, which is intentional.",
+  "description": "What is it: Suppresses Windows Firewall alerts to the end user for Domain and Private network firewalls. \n Simeon Notes: This policy improves user experience by suppressing warnings that the Domain and Private network firewall profiles allow inbound traffic by default.",
```

Comment:
**What does this do?**
Disables Windows Firewall alerts to the end user for Domain and Private network firewalls.

**Why should you use this?**
...

**What is the end-user impact?**
...

**Learn more**
N/A

@metamenos do we want to keep this? Jeff feels like anything that's questionable, when we're going against the Microsoft recommendation, shouldn't really be in the baseline. But what're your thoughts?